prisma/seed.ts

```diff
@@ -0,0 +1,87 @@
+import prisma from '../prisma/index'; // Relative path required, path aliases throw error with seed command
+import { Role } from '../lib/graphql/types'; // Relative path required, path aliases throw error with seed command
+
+const main = async () => {
+  if (process.env.NODE_ENV === 'production') {
+    await prodSeed();
+  } else {
+    await devSeed();
+  }
+};
+
+const prodSeed = async () => {
+  return;
```

Comment:
@OustanDing Wasn't sure what to populate the prod db with so the `prodSeed`  function doesn't do anything atm. I noticed in the (dancefest seed file)[https://github.com/uwblueprint/dancefest-web/blob/staging/prisma/seed.js] they seeded their prod admin table with one member from the team, lmk if you want me to do the same!

Reply:
Could you please just add me as the sole prod user? Thanks!

Reply:
Done! Also did some minor refactoring to not repeat `devSeed` code